_issues/but-hillary-wants-my-guns.md

```diff
@@ -4,32 +4,22 @@ title: "But Hillary wants my guns!"
 categories: guns, gun control
 ---
 
-Q: But Hillary wants my guns!
+Q: Doesn't Hillary want to take away my right to own a gun?
 
 
-A: That's not precisely correct!
+A: Hillary respects law-abiding citizens’ 2nd Amendment right to own guns. She just wants to put in place [common-sense solutions](https://www.hillaryclinton.com/issues/gun-violence-prevention/) to make our communities safer. 
 
-Basically, she doesn't want your guns, unless you're a felon, a
-domestic abuser, or someone with severe mental illnesses. And I think
-we can all agree that people like that shouldn't really have access to
-guns.
+Guns kill [around 33,000](https://en.wikipedia.org/wiki/Gun_violence_in_the_United_States) Americans a year. Hillary is committed to reducing the epidemic of gun violence. 
 
+Q: What do those solutions look like?
 
-Here's what her campaign [website](https://www.hillaryclinton.com/issues/gun-violence-prevention/) says:
+A: Here are the facts. As president, Hillary will:
+
++ __Expand background checks.__ A [large majority](http://fivethirtyeight.com/features/most-americans-agree-with-obama-that-more-gun-buyers-should-get-background-checks/) of Americans support expanding background checks for gun purchases. Currently, individuals can buy guns at gun shows or on the internet without a background check. Hillary will fight to pass legislation that closes these loopholes, as well as the [“Charleston Loophole,”](https://www.thetrace.org/2016/06/one-year-later-and-the-loophole-that-let-the-charleston-shooter-buy-his-gun-remains-wide-open/) which allows gun buyers to avoid a background check if the seller fails to complete the check within three days. 
```

Comment:
Perhaps better than referencing the Charleston Loophole would be to refer to the incident itself, e.g. "the loophole which allowed a shooter to gun down nine people in Charleston, South Carolina."